tests/test_transformation/test_arcsin_transformer.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+from sklearn.exceptions import NotFittedError
+
+from feature_engine.transformation import ArcsinTransformer
+
+
+def test_fit_raises_error_if_na_in_df(df_na):
+    # test case 2: when dataset contains na, fit method
+    with pytest.raises(ValueError):
+        transformer = ArcsinTransformer()
+        transformer.fit(df_na)
```

Comment:
can we add a few lines here to test when there is NA in the data to transform? We would have to fit in a df without na first